resources/js/humhub.wiki.Page.js

```diff
@@ -134,6 +134,11 @@ humhub.module('wiki.Page', function (module, require, $) {
             });
             $li.append($anchor);
             $list.append($li);
+
+            // Added numbering to content
+            if(document.querySelector('.numbered')){
+                $(this).prepend('<span class="wiki-header-numbering">' + numberString + '</span>');
```

Comment:
Please add a blank between the heading and the index.
```suggestion
                $(this).prepend('<span class="wiki-header-numbering">' + numberString + ' </span>');
```

Reply:
As just discussed with @JuergStae it even looks more structured if a space in combination with dot is used:
```suggestion
                $(this).prepend('<span class="wiki-header-numbering">' + numberString + '. </span>');
```